### DISCUSSION TITLE
Controller preset for X Touch One anyone??

### DISCUSSION THREAD

**Discussion by tompad2** [helgoboss/helgobox#472] [Q&A] [answered]
Hi!

I have a Behringer X Touch One I want to use with ReaLearn and instead of going thru with adding mappings for every function I wonder if anyone of you have a preset for XTO?

**Comment by helgoboss** [comment, marked as answer]
I was under the impression that I already published my X-Touch One controller preset but looks like I haven't. However, since it conforms to a Mackie UC device (at least in MCU mode), you can just try "Mackie Control" controller preset. And if you want it to actually do something, you can combine it with the "DAW Control" main preset.

**Reply by tompad2** [reply]
Yes, I have tested the MCU mode with DAW control, but no info of bars and beats showed in the LCD so I thought that maybe a XTO preset should fix that. But as you said in Reaper forum this function will come later in ReaLearn 2.11.

**Reply by helgoboss** [reply]
Hey @tompad2. You can now check out ReaLearn v2.11.0-pre.13: "Mackie Control" controller preset in combination with the "DAW control" main preset! After updating presets and ReaLearn via ReaPack, don't forget to reselect the presets in existing ReaLearn instances (as updating a preset doesn't automatically affect existing ReaLearn instances - a feature to prevent undesrred side effects through updates).

Of course, the main preset "DAW control" shows only one of infinite possibilities to make use of your X-Touch One. Feel free to adjust it to your needs.

**Comment by tompad2** [comment]
Great!

Will test it! 
I am really exited about this, yes you are right, there is infinite possibilities!
Great work Benjamin!

**Comment by tompad2** [comment]
ALRIGHT!!!

It works (LED bars timecode etc)! The only nagging I have is that it take some time to sync beats and bars when hitting play.
But its not a big deal for me - I dont sit and stir on the display when I hit play.

Love that F-buttons is free for my own suggestions! And the possibility to have double-press function on them, that
way I can have 2 actions on every F-button.

By the way - I am working on some lyrics for a song......they sound like this:

Bye, bye, bye, bye CSI! :-)

.....its a hit, isnt it? ;-)

**Reply by helgoboss** [reply]
> It works (LED bars timecode etc)! The only nagging I have is that it take some time to sync beats and bars when hitting play.
> But its not a big deal for me - I dont sit and stir on the display when I hit play.

Could you describe that in more detail or make little video of what you mean?

> By the way - I am working on some lyrics for a song......they sound like this:
> 
> Bye, bye, bye, bye CSI! :-)
> 
> .....its a hit, isnt it? ;-)

;)

**Reply by tompad2** [reply]
Let say that I start from bar 1 beat 1 - when I hit play on XT One Reaper starts to play directly, so Reaper responds
fast. But when watching the LED on XT One with numbers for bars and beats it dont start direct to count, its a delay 
before it responds with a jump and the counting catch up with Reapers counting.

Does my explaining make any sense?

**Reply by helgoboss** [reply]
Interesting, this doesn't happen here.

Windows or macOS? Could you send a test project and your reaper.ini?

**Reply by tompad2** [reply]
Realy??

I got Windows 10, and I use an empty project with just a few tracks, just for testing
ReaLearn and map my controllers. I have 3 instances of ReaLearn in monitorFX....
...hm, they are the last plugins in FX chain...can that be the reason?

I will make a test project and send reaper.ini - stand by.

**Reply by tompad2** [reply]
Nevermind!

Must be a temporary flaw in my setup/brain. Now when I click play it works!
Sorry! Its so strange, my setup is playing tricks with me...
